### PR TITLE
 Show the container runtime when running without kubernetes #13432 

### LINKED ISSUE #13432: Show the container runtime when running without kubernetes
Currently we hide the container runtime output.

But there should be plenty of room to show it ?

No Kubernetes

```
😄  minikube v1.25.1 on Ubuntu 20.04
✨  Automatically selected the kvm2 driver. Other choices: virtualbox, ssh
👍  Starting minikube without Kubernetes minikube in cluster minikube
🔥  Creating kvm2 VM (CPUs=2, Memory=3900MB, Disk=20000MB) ...
🏄  Done! minikube is ready without Kubernetes!
╭───────────────────────────────────────────────────────────────────────────────────────╮
│                                                                                       │
│                       💡  Things to try without Kubernetes ...                        │
│                                                                                       │
│    - "minikube ssh" to SSH into minikube's node.                                      │
│    - "minikube docker-env" to point your docker-cli to the docker inside minikube.    │
│    - "minikube image" to build images without docker.                                 │
│                                                                                       │
╰───────────────────────────────────────────────────────────────────────────────────────╯
```

With Kubernetes

```
😄  minikube v1.25.1 on Ubuntu 20.04
✨  Automatically selected the kvm2 driver. Other choices: virtualbox, ssh
👍  Starting control plane node minikube in cluster minikube
🔥  Creating kvm2 VM (CPUs=2, Memory=3900MB, Disk=20000MB) ...
🐳  Preparing Kubernetes v1.23.1 on Docker 20.10.12 ...
    ▪ kubelet.housekeeping-interval=5m
    ▪ Generating certificates and keys ...
    ▪ Booting up control plane ...
    ▪ Configuring RBAC rules ...
🔎  Verifying Kubernetes components...
    ▪ Using image gcr.io/k8s-minikube/storage-provisioner:v5
🌟  Enabled addons: default-storageclass, storage-provisioner
🏄  Done! kubectl is now configured to use "minikube" cluster and "default" namespace by default
```

----

This sentence is somewhat weird: "Starting minikube without Kubernetes minikube"

~Probably better off as "Starting virtual machine minikube" (or "container", for KIC) ?~

EDIT: It did say "VM" on the line below, so maybe something else rhyming with node.

```
👍  Starting node minikube in cluster minikube
🔥  Creating kvm2 VM (CPUs=2, Memory=3900MB, Disk=20000MB) ...
```

And this one doesn't contain much info: "Done! minikube is ready without Kubernetes!"

~Slightly more useful as "Done! minikube is ready with Docker 20.10.12" instead~

EDIT: Possibly on a separate line, to show the fancy runtime emoji (like that whale)

```
🐳  Preparing Docker 20.10.12 ...
🏄  Done! minikube is ready without Kubernetes!
```

### PR DESCRIPTION
**Pull request by Gimb0**
fixes #13432
In summary it is to show the runtime used when --no-kubernetes option is used to start minikube.
Also improves the readability of the sentence "Starting minikube without Kubernetes {machine name} in cluster {cluster}" by removing the machine name.

Changes to UI
Before:
image

After:
image

### CODE REVIEW

**Comment by Gimb0** [comment]
@afbjorklund - Can you please review this PR and give me any feedback you may have?

**Review by afbjorklund — APPROVED** [review]
Output (and code) looks good to me, thanks for fixing!

File: pkg/minikube/node/config.go [hunk 1]
```diff
@@ -51,6 +51,11 @@ func showVersionInfo(k8sVersion string, cr cruntime.Manager) {
 	}
 }
 
+func showNotK8sVersionInfo(cr cruntime.Manager) {
```
Comment (spowelljr) [inline pkg/minikube/node/config.go:54]:
```suggestion
func showNoK8sVersionInfo(cr cruntime.Manager) {
```
nit

**Comment by sharifelgamal** [comment]
There's also a lint issue: `pkg/minikube/node/start.go:105: File is not `gofmt`-ed with `-s` (gofmt)`

File: pkg/minikube/node/config.go [hunk 2]
```diff
@@ -51,6 +51,11 @@ func showVersionInfo(k8sVersion string, cr cruntime.Manager) {
 	}
 }
 
+func showNoK8sVersionInfo(cr cruntime.Manager) {
+	version, _ := cr.Version()
```
Comment (spowelljr) [inline pkg/minikube/node/config.go:55]: I don't think we should be ignoring this error, at the least we should be logging a `klog.Warningf` so we have it logged somewhere that that command failed.

**Review by spowelljr — CHANGES_REQUESTED** [review]

File: pkg/minikube/node/config.go (hunk 2 above)
Comment (Gimb0) [reply pkg/minikube/node/config.go]: I've made the changes to capture the error and added a `klog.Warningf` if it returns an error.

File: pkg/minikube/node/start.go [hunk 3]
```diff
@@ -97,7 +97,14 @@ func Start(starter Starter, apiServer bool) (*kubeconfig.Settings, error) {
 	}
 	if stopk8s {
 		nv := semver.Version{Major: 0, Minor: 0, Patch: 0}
-		configureRuntimes(starter.Runner, *starter.Cfg, nv)
+		cr := configureRuntimes(starter.Runner, *starter.Cfg, nv)
+
+		if err = cruntime.CheckCompatibility(cr); err != nil {
```
[the remark this answers is not in the document]
Comment (Gimb0) [reply pkg/minikube/node/start.go]: I've made some changes based on your comment. Can you please review @spowelljr ?

**Review by spowelljr — APPROVED** [review]
Thanks for the PR

**Pull request merged**